vcpkg/consume/lock-package-versions.md

```diff
@@ -0,0 +1,238 @@
+      "baseline": "1.2.11",
+      "port-version": 9
+    },
+```
+
+## 3 - Update the baseline versions
+
+Baselines offer a convenient mechanism to update the versions of all your dependencies at once. To
+update your baseline run the following command:
+
+```bash
+vcpkg x-update-baseline
+```
+
+The [`x-update-baseline`](../commands/update-baseline.md) command modifies your manifest file to set
+[`builtin-baseline`](../reference/vcpkg-json.md#builtin-baseline) to the current Git commit of your
+vcpkg instance.
+
+You can use the `--add-initial-baseline` option to add a `builtin-baseline` to a manifest that does
+not have one yet.
+
+## 4 - Add a minimum version constraint
+
+Baselines are not the only way to lock down a package's version. vcpkg also accepts minimum version
+constraints in the form of `version>=`.
+
+Modify the contents of `vcpkg.json` to:
+
+```json
+{
+  "dependencies": [
+    {
+        "name": "fmt",
+        "version>=": "10.1.1"
+    },
+    "zlib"
+  ],
+  "builtin-baseline": "3426db05b996481ca31e95fff3734cf23e0f51bc"
+}
+```
+
+The manifest file above uses the [dependency object](../reference/vcpkg-json.md#dependency-fields)
+notation to set a minimum version constraint (`version>=`) on `fmt`. In order to satisfy the
+dependencies vcpkg needs to satisfy two constraints, one comfing from the baseline and one coming
+from the minimum version constraint in the `dependencies` list.
+
+* Baseline constraint, `"version>=": "7.1.3"`.
+* Dependencies list constraint, `"version>=": "10.1.1"`.
+
+Build and run the project, replace `%VCPKG_ROOT%` with your vcpkg installation path:
+
+```bash
+rm -r build
+cmake -B build -S . -DCMAKE_TOOLCHAIN_FILE=%VCPKG_ROOT%/scripts/buildsystems/vcpkg.cmake
+cmake --build build
+./build/main
+```
+
+The output should look like this:
+
+```Console
+fmt version is 100100
+zlib version is 1.2.11
+```
+
+In this case, version `10.1.1` of `fmt` satisfies both constraints. Notice how `zlib` gets its
+baseline version `1.2.11`.
+
+## 5 - Force a specific version
+
+In some cases, you may want to force a specific version of a package, for example: 
+
+* To resolve version conflicts.
+* To lock down versions older than the baseline.
+* To lock down versions that are otherwise incomparable, for example: `vista`, `xp`. 
+
+vcpkg allows you to solve these issues by using version overrides.
+
+Modify the `vcpkg.json` contents to:
+
+```json
+{
+  "dependencies": [
+    {
+        "name": "fmt",
+        "version>=": "10.1.1"
+    },
+    "zlib"
+  ],
+  "builtin-baseline": "3426db05b996481ca31e95fff3734cf23e0f51bc", 
+  "overrides": [
+    { 
+        "name": "zlib", 
+        "version": "1.2.8"
+    }
+  ]
+}
+```
+
+Any package included in the `"overrides"` list will use the specified version while ignoring all
+other version constraints. In this example, the baseline `3426db05b996481ca31e95fff3734cf23e0f51bc`
+adds a `"version>=": "1.2.11"` constraint on `zlib` but the override declaration forces version
```

Comment:
```suggestion
adds a minimum version constraint on `zlib` of `1.2.11` but the override declaration forces version
```

We shouldn't use the exact syntax `"version>="` for baselines since they are not equivalent. You can have baselines without `version>=`, but you can't have `version>=` without baselines.